File: app/src/main/java/me/pwcong/jpstart/network/baidu/service/BaiduService.java
package me.pwcong.jpstart.network.baidu.service;

import me.pwcong.jpstart.mvp.bean.BaiduTranslateBean;
import rx.Subscriber;

/**
 * Created by Pwcong on 2016/10/1.
 */

public interface BaiduService {

    interface TranslateService {

        void translate(String q, String from, String to, Subscriber<BaiduTranslateBean> subscriber);

    }


}

File: app/src/main/java/com/github/pwcong/jpstart/network/baidu/service/BaiduService.kt
package com.github.pwcong.jpstart.network.baidu.service

import com.github.pwcong.jpstart.mvp.bean.BaiduTranslateBean
import rx.Subscriber

interface BaiduService {
    interface TranslateService {
        fun translate(
            q: String,
            from: String,
            to: String,
            subscriber: Subscriber<BaiduTranslateBean>
        )
    }
}
